src/Template/Layout/js/app/pages/modules/view.js

```diff
@@ -34,5 +34,25 @@ export default {
                 return this.tabsOpen = !this.tabsOpen;
             }
         },
+
+        fetchAllTranslations() {
+            this.$helpers.autoTranslate('<b>i will eat</b> an <span>aubergine</span>', 'en', 'it');
```

Comment:
Should  `fetchAllTranslations` really work like this? :smiley:

Reply:
this was temporary. incoming patch...